iib/web/models.py

```diff
@@ -698,6 +704,15 @@ def _from_json(
                     ' the "overwrite_from_index_token" parameter is used'
                 )
 
+        distribution_scope = request_kwargs.pop('distribution_scope', None)
+        if distribution_scope:
+            distribution_scope = distribution_scope.upper()
+            if distribution_scope not in ['PROD', 'STAGE', 'DEV']:
```

Comment:
Somewhat optional: Rather than hard-code these distribution scopes, I'd rather see this be configurable in the event more distribution scopes are added or renamed.

Reply:
I think that we want to make the entire thing configurable in a follow on IMO. 

What I mean is the label key, potential values and this bit here too, we want to make a configuration value IMO.

WDYT of that?

Reply:
I agree with the approach but I'd like to see it in this MR if possible. I'll leave it up to @yashvardhannanavati to decide on that though.

Reply:
Would that involve building the plumbing to pass configuration through to the workers though? As well as designing what to capture in the configuration to define the semantics of each value.

Reply:
I think we can duplicate the configuration in the API and the workers since this happens infrequently.